src/Extensions/DependencyInjection/FunctionsHostBuilderContext.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.Functions.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Context containing the common services and settings for the Functions host.
+    /// </summary>
+    public abstract class FunctionsHostBuilderContext
+    {
+        private readonly WebJobsBuilderContext _webJobsBuilderContext;
+
+        public FunctionsHostBuilderContext(WebJobsBuilderContext webJobsBuilderContext)
```

Comment:
Is this being renamed from the previous changes?

Reply:
Yes it is called `FunctionsHostBuilder` (with an 's') yet this was called `FunctionHostBuilderContext`... so I added the 's' to match it.